core/src/logging.rs

```diff
@@ -32,14 +32,21 @@ pub fn init(filter: impl AsRef<str>) -> (Logger, GlobalLoggerGuard) {
 }
 
 /// Sets a panic hook so panic information is written with the log facilities
-/// instead of directly to STDERR.
+/// in addition to the default panic printer.
 fn set_panic_hook() {
-    fn hook(info: &PanicInfo) {
+    let default_hook = panic::take_hook();
+    let hook = move |info: &PanicInfo| {
         let thread = thread::current();
         let thread_name = thread.name().unwrap_or("<unnamed>");
 
+        // It is not possible for our custom hook to print a full backtrace on stable rust. To not
+        // lose this information we call the default panic handler which prints the full backtrace.
+        // We print a fake log message prefix so that kibana can identify that this is supposed to
+        // a single message.
+        eprint!("{} ERRO [core::logging] ", formatted_current_time());
```

Comment:
any chance we can reuse the `log_to_decorator` method? At least, I would say we should format the log level so we are bit more future proof:
```suggestion
        eprint!("{} {} [core::logging] ", formatted_current_time(), log::Level::Error);
```

Reply:
I think we should be able to use [`PlainDecorator<Vec<u8>>`](https://docs.rs/slog-term/2.6.0/slog_term/struct.PlainDecorator.html)

Reply:
implemented